### PR TITLE
Fix pep8 errors in viz

### LINKED ISSUE #890: PEP8 in viz
dipy/viz/colormap.py:152:18: E222 multiple spaces after operator
dipy/viz/fvtk.py:4:80: E501 line too long (136 > 79 characters)
dipy/viz/fvtk.py:5:80: E501 line too long (120 > 79 characters)
dipy/viz/fvtk.py:287:80: E501 line too long (86 > 79 characters)
dipy/viz/fvtk.py:295:80: E501 line too long (114 > 79 characters)
dipy/viz/fvtk.py:440:80: E501 line too long (155 > 79 characters)
dipy/viz/fvtk.py:441:80: E501 line too long (112 > 79 characters)
dipy/viz/fvtk.py:475:80: E501 line too long (96 > 79 characters)
dipy/viz/fvtk.py:676:29: E231 missing whitespace after ':'
dipy/viz/fvtk.py:676:47: E231 missing whitespace after ':'
dipy/viz/fvtk.py:677:1: E302 expected 2 blank lines, found 0
dipy/viz/fvtk.py:910:32: E127 continuation line over-indented for visual indent
dipy/viz/fvtk.py:917:80: E501 line too long (80 > 79 characters)
dipy/viz/projections.py:17:1: E402 module level import not at top of file
dipy/viz/projections.py:19:1: E402 module level import not at top of file
dipy/viz/projections.py:48:80: E501 line too long (81 > 79 characters)
dipy/viz/projections.py:59:80: E501 line too long (89 > 79 characters)
dipy/viz/projections.py:79:46: E231 missing whitespace after ','
dipy/viz/projections.py:79:48: E231 missing whitespace after ','
dipy/viz/projections.py:79:52: E231 missing whitespace after ','
dipy/viz/projections.py:79:55: E231 missing whitespace after ','
dipy/viz/projections.py:79:57: E231 missing whitespace after ','
dipy/viz/projections.py:79:60: E231 missing whitespace after ','
dipy/viz/projections.py:79:63: E231 missing whitespace after ','
dipy/viz/projections.py:79:65: E231 missing whitespace after ','
dipy/viz/projections.py:79:80: E501 line too long (81 > 79 characters)
dipy/viz/projections.py:81:80: E501 line too long (80 > 79 characters)
dipy/viz/projections.py:82:36: E225 missing whitespace around operator
dipy/viz/projections.py:105:9: E122 continuation line missing indentation or outdented
dipy/viz/projections.py:105:51: E231 missing whitespace after ','
dipy/viz/projections.py:106:51: E231 missing whitespace after ','
dipy/viz/projections.py:107:65: E231 missing whitespace after ','
dipy/viz/projections.py:113:12: E225 missing whitespace around operator
dipy/viz/projections.py:113:15: E225 missing whitespace around operator
dipy/viz/projections.py:114:12: E225 missing whitespace around operator
dipy/viz/projections.py:114:15: E225 missing whitespace around operator
dipy/viz/projections.py:116:44: E231 missing whitespace after ','
dipy/viz/projections.py:116:46: E231 missing whitespace after ','
dipy/viz/projections.py:131:40: E261 at least two spaces before inline comment
dipy/viz/projections.py:134:1: W391 blank line at end of file
dipy/viz/regtools.py:20:1: E302 expected 2 blank lines, found 1
dipy/viz/regtools.py:68:23: E127 continuation line over-indented for visual indent
dipy/viz/regtools.py:76:1: E302 expected 2 blank lines, found 1
dipy/viz/regtools.py:281:80: E501 line too long (81 > 79 characters)
dipy/viz/regtools.py:302:23: E127 continuation line over-indented for visual indent
dipy/viz/regtools.py:303:23: E127 continuation line over-indented for visual indent
dipy/viz/regtools.py:311:1: E302 expected 2 blank lines, found 1
dipy/viz/regtools.py:385:23: E127 continuation line over-indented for visual indent
dipy/viz/regtools.py:386:23: E127 continuation line over-indented for visual indent
dipy/viz/widget.py:21:26: E251 unexpected spaces around keyword / parameter equals
dipy/viz/widget.py:21:28: E251 unexpected spaces around keyword / parameter equals
dipy/viz/widget.py:136:80: E501 line too long (80 > 79 characters)
dipy/viz/widget.py:137:80: E501 line too long (83 > 79 characters)
dipy/viz/widget.py:221:80: E501 line too long (80 > 79 characters)
dipy/viz/widget.py:222:80: E501 line too long (83 > 79 characters)
dipy/viz/widget.py:318:80: E501 line too long (80 > 79 characters)
dipy/viz/widget.py:319:80: E501 line too long (83 > 79 characters)
dipy/viz/tests/__init__.py:5:11: W292 no newline at end of file
dipy/viz/tests/test_regtools.py:34:50: E251 unexpected spaces around keyword / parameter equals
dipy/viz/tests/test_regtools.py:34:52: E251 unexpected spaces around keyword / parameter equals

### PR DESCRIPTION
**Pull request by ghoshbishakh**
fixes #890

### CODE REVIEW

**Comment by arokem** [comment]
Thanks for contributing. I don't think you did anything wrong. I believe the failures on the Travis bot are due to updates in our dependencies. I'll post an issue about this, but we'll hold off on merging this in, until we can sort that out.

File: dipy/viz/projections.py [hunk 1]
```diff
@@ -56,7 +55,8 @@ def sph_project(vertices, val, ax=None, vmin=None, vmax=None, cmap=None,
     --------
     >>> from dipy.data import get_sphere
     >>> verts = get_sphere('symmetric724').vertices
-    >>> ax = sph_project(verts.T, np.random.rand(len(verts.T))) # skip if not has_basemap
+    >>> ax = sph_project(verts.T, np.random.rand(len(verts.T))) # skip
```
Comment (arokem) [inline dipy/viz/projections.py]: Sorry - I was wrong. This change is the change that causes the test failures in the bots. Please change this line back. 

This causes Travis to skip this test, when basemap is not present, and the Travis set up doesn't (usually) have this package.

**Comment by ghoshbishakh** [comment]
It is still inside the comment right? Changing it back gives the pep8 line too long. So can I solve it somehow?

**Comment by arokem** [comment]
Nope. Sometimes we just have to live with PEP8 violations: https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

**Comment by arokem** [comment]
In it goes. Thanks for the fix!

**Comment by ghoshbishakh** [comment]
thanks, Ill get back at the other pep8 s :)

**Pull request merged**